atomic.conf

```diff
@@ -3,3 +3,4 @@
 default_scanner: openscap
 default_docker: docker
 
+# default_storage: ostree
```

Comment:
this is the default when nothing is set as well.  Do you want it to be "default" in the sense of being explicit in the configuration file or also when nothing is set?
At the moment only `ostree` storage is supported.